doc/api/index.md

```diff
@@ -35,7 +34,8 @@
 * [HTTPS](https.html)
 * [Inspector](inspector.html)
 * [Internationalization](intl.html)
-* [Modules](modules.html)
+* [Modules: CommonJS modules](modules.html)
+* [Modules: ECMAScript modules](esm.html)
 * [Modules: `module` core module](module.html)
```

Comment:
There has to be a better way to title this. This is incredibly hard to understand. (Sorry I didn't get that comment in before it landed.)

Reply:
Maybe this?

```suggestion
* [Modules: shared object](module.html)
```

Or this?

```suggestion
* [Modules: utility object](module.html)
```

Or even:

```suggestion
* [Modules: shared utility object](module.html)
```

Reply:
It's not an object, it's an API like `fs` or `process`. Perhaps “Modules: `module` API”?

Reply:
```suggestion
* [`module` API](module.html)
```

It's probably OK if it doesn't appear next to the other ones?

Reply:
I would go with “Modules: `module` API”.